integreat_cms/cms/models/abstract_content_translation.py

```diff
@@ -568,6 +572,48 @@ def save(self, *args: Any, **kwargs: Any) -> None:
             self.last_updated = timezone.now()
         super().save(*args, **kwargs)
 
+    @transaction.atomic
+    def cleanup_autosaves(self, keep=3):
+        """
+        Delete all but the last ``keep`` AUTO_SAVEs
+        and renumber all affected versions to be continuous.
+
+        :param keep: How many auto saves should be kept
+        :type keep: int
+        """
+        logger.debug("Cleaning up autosaves except the latest %s ones.", keep)
+        delete_auto_saves = self.foreign_object.translations.filter(
+            language=self.language, status=status.AUTO_SAVE
+        )[keep:]
```

Comment:
```suggestion
        delete_auto_saves = list(
            self.foreign_object.translations.filter(
                language=self.language, status=status.AUTO_SAVE
            )[keep:]
        )
```
Due to Django's restrictions with sliced querysets, I'd suggest to convert this to a list, which makes it easier to use the results of the queryset without having to make new queries.